package/libs/wolfssl/Makefile

```diff
@@ -77,12 +79,35 @@ define Package/libwolfssl/config
 	source "$(SOURCE)/Config.in"
 endef
 
+define Package/libwolfssl-cpu-crypto
+$(call Package/libwolfssl/Default)
+  TITLE:=wolfSSL library with AES CPU instructions
+  PROVIDES:=libwolfssl libcyassl
+  DEPENDS:=@(aarch64||x86_64) @!bcm27xx
```

Comment:
I think the `libwolfssl-cpu-crypto` package will not get build if the SDK for `bcm27xx` is used to build the shared packages. Which SDK do we use to build the shared package in our build bots?

I think it would be saver if we remove the `@!bcm27xx`

Reply:
I'm only allowing it to be built as a module on bcm27xx.
Tell me if I'm getting too far out of the way trying to avoid building a broken image, and I'll keep it simple.

Reply:
> Which SDK do we use to build the shared package in our build bots?

Looking at the code alone, I'm assuming that the first target/subtarget pair from `./scripts/dump-target-info.pl architectures` is used to build for each architecture.  So aarch64_cortex-a72 will be built with bcm27xx/bcm2711 SDK:
```
$ ./scripts/dump-target-info.pl architectures 2>/dev/null | grep aarch64
aarch64_cortex-a53 armvirt/64 bcm27xx/bcm2710 bcm4908/generic mediatek/mt7622 mediatek/filogic mvebu/cortexa53 sunxi/cortexa53
aarch64_cortex-a72 bcm27xx/bcm2711 mvebu/cortexa72
aarch64_generic layerscape/armv8_64b octeontx/generic rockchip/armv8
```